contract/contracts/AstarDegens.sol

```diff
@@ -1248,13 +1248,13 @@ contract AstarDegens is ERC721Enumerable, Ownable {
   function mint(uint256 _mintAmount) public payable {
     uint256 supply = totalSupply();
     require(!paused, "Degens are lazy, call the king to wake them up");
-    require(_mintAmount > 0);
+    require(_mintAmount > 0, "Mint amount is 0");
     require(_mintAmount <= maxMintAmount, "Degen tribe is max 5 apes");
```

Comment:
You should remove this two checks and just replace them with what you have on line 1257.

Reply:
let's discuss. I do not agree